docs/install.rst

```diff
@@ -35,7 +35,7 @@ The easiest way to install Python and SciPy is with `Anaconda`_, a free scientif
 
 #. We can install Pylake in this environment by invoking the following command::
 
-    conda install lumicks.pylake
+    conda install lumicks.pylake jupyter_client=7.4.9
```

Comment:
should we motivate this somehow? it seems kind of odd to tell a user to install a specific version of a package with no reasoning

Reply:
True. I've added a link to the open issue. Also nice to just have it around so we can remove this instruction again when it's fixed upstream.